package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "autod": "autod",
     "lint": "eslint lib test *.js",
```

Comment:
change to `eslint .`

Reply:
What does this mean? Do you mean checking all kinds of files or the shorten symbol of *.js?